From ddc1b6b559c220700869e7e7761b26b0f9e84d9f Mon Sep 17 00:00:00 2001
From: Wolfgang Bauer <wbauer@tmo.at>
Date: Tue, 30 May 2017 20:24:19 +0200
Subject: [PATCH 1/2] Fix build with Qt 5.9

moc 5.9 errors out when building:
Error: Plugin Metadata file "" could not be opened: file to open is a
directory

Same issue and fix as https://phabricator.kde.org/D5392 for khtml.

CCBUG: 377490
Differential Revision: https://phabricator.kde.org/D5985
---
 src/kmplayer_part.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/kmplayer_part.h b/src/kmplayer_part.h
index f90f85d..0fddbaa 100644
--- a/src/kmplayer_part.h
+++ b/src/kmplayer_part.h
@@ -36,7 +36,7 @@ namespace KMPlayer {
 
 class KMPlayerFactory : public KPluginFactory {
     Q_OBJECT
-    Q_PLUGIN_METADATA(IID "org.kde.KPluginFactory" FILE "")
+    Q_PLUGIN_METADATA(IID "org.kde.KPluginFactory")
     Q_INTERFACES(KPluginFactory)
 public:
     KMPlayerFactory();
-- 
2.12.3

